server/resource/caldav_service.go

```diff
@@ -1,18 +1,545 @@
 package resource
 
 import (
+	"database/sql"
+	"encoding/base64"
+	"fmt"
+	"github.com/daptin/daptin/server/database"
+	"github.com/jmoiron/sqlx"
 	"github.com/samedi/caldav-go"
+	"github.com/samedi/caldav-go/data"
+	"github.com/samedi/caldav-go/errs"
+	log "github.com/sirupsen/logrus"
 	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
 )
 
-func NewCaldavServer(CaldavListenInterface string)*http.Server{
+var Schema = `
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS calendar;
+DROP TABLE IF EXISTS users;
+DROP TABLE IF EXISTS collection;
+DROP TABLE IF EXISTS collection_role;
+
+
+CREATE TABLE calendar (
```

Comment:
these tables to be to the StandardTables list. But the expectation here is to have it as a row in document, like mentioned earlier in the ui implementation.